src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -360,6 +390,18 @@ class ActivityFeed extends React.Component<Props, State> {
     } = this.state;
 
     const feedList = getSortedFeedData(tabs, activeTab, feedData);
+    const feedSections = [];
+
+    feedList.forEach((listItem) => {
+      const formattedDate = formatDate(new Date(listItem.createdAt * 1000), 'MMM D');
+      const existingSection = feedSections.find(({ title }) => title === formattedDate) || {};
```

Comment:
Do we really need the default object here? I think it would be cleaner to just use the undefined return from `find`:

```js
const existingSection = feedSections.find(({ title }) => title === formattedDate);
if (existingSection) {
  existingSection.data.push(listItem);
} else {
  feedSections.push({ title: formattedDate, data: [{ ...listItem }] });
}
```